integration-tests/config.ts

```diff
@@ -80,45 +80,45 @@ const hangzhounetEphemeral = {
   signerConfig: {
     type: SignerType.EPHEMERAL_KEY as SignerType.EPHEMERAL_KEY,
     keyUrl: 'https://api.tez.ie/keys/hangzhounet',
-    requestHeaders: { 'Authorization': 'Bearer taquito-example' },
-  }
-}
+    requestHeaders: { Authorization: 'Bearer taquito-example' },
+  },
+};
 
-const idiazabalnetFaucet = {
-  rpc: process.env['TEZOS_RPC_IDIAZABALNET'] || 'https://idiazabalnet.ecadinfra.com',
+const ithacanetFaucet = {
+  rpc: process.env['TEZOS_RPC_ITHACANET'] || 'https://ithacanet.ecadinfra.com/',
   knownBaker: 'tz1cjyja1TU6fiyiFav3mFAdnDsCReJ12hPD',
-  knownContract: 'KT1UUS3M31GpFS4k5xQtp97oWGDhdzdT9mNJ',
-  knownBigMapContract: 'KT1VFM8dyKv2YTFdLBZm8seJKgFnn7RMrDtv',
-  knownTzip1216Contract: 'KT1PiiNe7icQHeaQ4Z8m27FVhYWQ2pPGMots',
-  protocol: Protocols.ProtoALpha,
+  knownContract: 'KT1J3BCZt5o8CA1xgqbV9Z1iEgoxuS1RamTa',
+  knownBigMapContract: 'KT1VC9dVmYYHyUXEw1KU8Ms6BmtvsGJNarZy',
+  knownTzip1216Contract: 'KT1DWbrEN9BzoutK5LYfSe9DWrt7D2aoaoTJ',
+  protocol: Protocols.PsiThaCa,
   signerConfig: {
     type: SignerType.FAUCET as SignerType.FAUCET,
-    faucetKey: {
-      "pkh": "tz1S6Y5J1i5JbXQf9eigvrZdghtjCApC8gUG",
+      faucetKey: {  
+      "pkh": "tz1LyuwEK3cPX8eHankqJW6hKJNpXAEPZaQz",
       "mnemonic": [
-        "throw",
-        "rose",
-        "girl",
-        "arrange",
-        "practice",
-        "fiber",
-        "speed",
-        "delay",
-        "economy",
-        "wine",
-        "cable",
-        "copper",
-        "sweet",
-        "bag",
-        "wasp"
+        "source",
+        "fantasy",
+        "diary",
+        "situate",
+        "spring",
+        "prize",
+        "solution",
+        "dawn",
+        "exotic",
+        "butter",
+        "barrel",
+        "garden",
+        "transfer",
+        "alcohol",
+        "edit"
       ],
-      "email": "hxkxykcy.edpwomka@teztnets.xyz",
-      "password": "vzVIvhuItq",
-      "amount": "143524244923",
-      "secret": "c83d1fba98e408bc142b3a9568a27207e5544b99"
-    },
-  }
-}
+      "email": "lcmxowve.lmlksvxa@teztnets.xyz",
+      "password": "rudDJKxp44",
+      "amount": "116492897916",
+      "activation_code": "0dfbf62414e7f10dbb47fc3b16cfd6b95d99c22c"
```

Comment:
We need to replace activation_code by secret

Reply:
replaced